server.js

```diff
@@ -8,6 +8,8 @@ const path = require('path')
 // Create our app
 const app = express()
 
+app.get('/config.json', (_, res) => res.sendFile(path.join(__dirname, 'config.json')))
```

Comment:
Can you instead make webpack copy this json file in the public folder (along with the rest of the assets) ?

Reply:
That will make it not really dynamic as pepole will have to rebuild for config to change and in docker it will not update unless you rebuild the whole image which defeats the point

Reply:
Ok I see so you can keep this line

You still need to add it to webpack otherwise it won't work on GitHub action